plugins/package-managers/gradle-inspector/README.md

```diff
@@ -0,0 +1,45 @@
+# GradleInspector
+
+The [GradleInspector] is an alternative analyzer for projects that use the Gradle package manager. It is supposed to
+address [several][] [shortcomings][] of the "legacy" [Gradle] analyzer, but in order to not interfere with it, the
```

Comment:
Is the `[]` required here? You don't use it for the other links.

Reply:
Yes, they're required here because the other links are not HTTP / external links.